dotnet3.5/S3/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
```

Comment:
Maybe add a comment toward the top of the code indicating what version of the AWS SDK for .NET and what version of .NET this code was developed against? If so, I would stick with listing major versions, otherwise these will need to be updated with every "dot" release!

Reply:
Oops, answered this above. Added that the example uses AWS .NET SDK 3.5 and .NET 5.